tests/test_pretest.py

```diff
@@ -157,9 +157,14 @@ def test_disable_rsyslog_rate_limit(duthosts, enum_dut_hostname):
         # We don't want to fail here because it's an util
         logging.warn("Failed to retrieve feature status")
         return
+    config_facts = duthost.config_facts(host=duthost.hostname, source="running")
+    is_dhcp_server_enable = config_facts["ansible_facts"]["DEVICE_METADATA"]["localhost"]["dhcp_server"]
```

Comment:
Please make sure all keys are presented in dict, otherwise it might throw key not found exception and crash the pretest

Reply:
Fixed